src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("GetAllProductsFromShoppingCart successful fetched data from DB");
```

Comment:
The logging message states that data was fetched from the DB, but the method currently returns hardcoded data. Ensure that this is updated to reflect actual data retrieval logic in a production environment.